### PR TITLE
Starting RN2232

### PR DESCRIPTION
**Pull request by levshamin**

### CODE REVIEW

File: content/_releases/22/32.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+redirect_from:
+  - /releases/22.32.html
+---
+
+
+## Improvements and Updates
+
+### Continued improvements
+
+Here we should talk about the automated vulnerabilities checks of our code.
```
Comment (KMobbs) [inline content/_releases/22/32.md:18]: As part of ongoing program of improving our code base and in-line with our ISO27001 processes we have recently completed a number of improvements to both address existing vulnerabilities and prevent potential vulnerabilities being released. Instrumental to this approach is the use of regular scans of our code base during back ups.

An high-level overview of our InfoSec standards and processes are available here (link to the high level infosec doc)

File: content/_releases/22/32.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+{% if site.data.tenant.name == "elastic.io" %}
+
+## HELM3
+
+We constantly improve different aspects of HELM3 deployments in collaboration
+with our partners. This section lists updates and improvements included in this release.
+
+### Scalability of services
+
+elastic.io platform runs many services under the hood like `webhooks`, `api`, `frontend`, etc.
```
Comment (KMobbs) [inline content/_releases/22/32.md:29]: ..hood, such as 'webhooks'..

File: content/_releases/22/32.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+Most of these services can and must run in replication meaning they can be scalled to suit
```
Comment (KMobbs) [inline content/_releases/22/32.md:30]: Most of these services can be replicated, which allows the services to be scaled horizontally to address the required load and volume. However, some platform services are inherently not scalable. In this update we removed the possibility to define scaling within HELM chart configurations for 'bloodygate' and 'handmaiden' services.

File: content/_releases/22/32.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+[OEM version of the elastic.io platform](https://www.elastic.io/saas-embedded-integration/).
+
+### Contract level custom metadata
+
+We introduce a new possibility to add a custom metadata to the contracts. A new
+section called `custom_data` can be used to store non-sensitive string data up-to 1MB in
+size. To add, edit and read contract custom metadata you need to have
+`contracts.contract.edit_custom_data` permission (`tenantAdmin` has this permission
+by default) to perform `POST`, `PATCH` and `GET` calls to the `/v2/contract` API endpoint.
+
```
Comment (KMobbs) [inline content/_releases/22/32.md:47]: This new feature enables Tenant Owners to better differentiate different types of Contracts they supply to their Clients and Users. For instance, a selection of different Contract types can be defined and automatically created, deployed, and maintained using this metadata as basis. Each contract type can have a wide range of specified features, such as GB RAM quota limits, numbers of users or workspaces, and even connector whitelists.

File: content/_releases/22/32.md (hunk 1 above)
Comment (khanzadyan) [reply content/_releases/22/32.md:18]: your are the best

File: content/_releases/22/32.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+{% endif %}
+
+## Fixed bugs
+
+*   We corrected the error type for the `PATCH` call to `/v2/flows/{FLOW_ID}` when invalid team name in the command parameter and invalid trigger/action name in the command parameter were given. Now you will get `4xx` type error instead of `5xx` which better reflects the situation.
```
Comment (KMobbs) [inline content/_releases/22/32.md:52]: ..when an invalid team name and (or??) an invalid trigger/action name are defined in the command parameter.

**Pull request merged**